File: RxJsoupNetWorkLibrary/src/main/java/io/reactivex/jsoup/network/bus/RxBusCallBack.java
package io.reactivex.jsoup.network.bus;

/**
 * by y on 2017/2/23
 */

public interface RxBusCallBack<T> {
    void onBusNext(T t);

    void onBusError(Throwable throwable);

    Class<T> busOfType();
}

File: RxNetWorkLibrary/src/main/java/io/reactivex/network/bus/RxBusCallBack.kt
package io.reactivex.network.bus

/**
 * by y on 2017/2/23
 */

interface RxBusCallBack<T> {
    fun onBusNext(t: T)

    fun onBusError(throwable: Throwable)

    fun busOfType(): Class<T>
}

